assets/_scss/components/_accordions.scss

```diff
@@ -1,51 +1,53 @@
 #accordion-menu {
-  border: 1px solid $color-grey-600;
-  border-radius: 5px;
-  color: $color-grey-900;
-  margin: 0;
-  overflow: hidden;
-  padding: 0;
-
-  li {
-    background-color: $color-grey-200;
-    border-bottom: 1px solid $color-grey-600;
-    font-family: $font-sans;
-    list-style: none;
+  ul {
```

Comment:
I thought this PR was just for adding functionality to the style guide itself, not the actual pattern library code. Why was this selector added?